tests/models/test_serialized_dag.py

```diff
@@ -39,33 +39,21 @@ def make_example_dags(module):
     return dagbag.dags
 
 
-def clear_db_serialized_dags():
-    with create_session() as session:
-        session.query(SDM).delete()
-
-
-@parameterized_class(
-    [
-        {"compress_serialized_dags": "False"},
-        {"compress_serialized_dags": "True"},
-    ]
-)
```

Comment:
This parameters always evaluate as `True`, so tests alway runs against compressed compress serialized DAGs.

https://github.com/apache/airflow/blob/2794662459b6d2f40b55b9b1d39eed14b5c3c5e9/airflow/models/serialized_dag.py#L107-L112

This fixed